ui/src/utils/index.tsx

```diff
@@ -40,6 +40,15 @@ export function validateGitHubPAT(token: string) {
   return /^ghp_[a-zA-Z0-9]{36}$/ig.test(token)
 }
 
+/**
+ * Method to validate given token is a right GitLab PAT
+ * @param token Token string to validate
+ * @returns true if token is a right GitLab PAT, otherwise return false
+ */
+export function validateGitLabPAT(token: string) {
+  return /^glpat-[a-zA-Z0-9]{36}$/ig.test(token)
```

Comment:
```suggestion
  return /^glpat-[a-zA-Z0-9]{20}$/ig.test(token)
```

I'm not sure if this is true of all tokens, but the one I created for testing was 20 characters after the `glpat-` prefix. If we don't find anything official in the docs about the length/size of the tokens, I think the only validation we should do is checking for the prefix - and maybe eventually an API call